extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -390,6 +390,14 @@ export class GoogleSubscriptionsPlatform {
         } else {
           return null;
         }
+        if (
```

Comment:
Hmm. This part seems to be repeated a lot. Can we find a way to move it upstairs? E.g. it'd look like this:

```
platform.getEntitlements().then(ents => {
  if (documentIsEncrypted && ents.granted && !ents.decryptedDocumentKey) {
    // log error
    return null;
  }
  return ents;
})
```

Reply:
Moved code into amp-subscriptions directly.